src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/EventAuditRepository.kt

```diff
@@ -11,7 +11,7 @@ import java.time.LocalDateTime
 
 @Repository
 interface EventAuditRepository : JpaRepository<EventAudit, Long> {
-  fun findByBookingReference(bookingReference: String): List<EventAudit>
```

Comment:
test were randomly failing,... because the order was incorrect, added this seems to have fixed it